components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/universal-deployer.adoc

```diff
@@ -0,0 +1,155 @@
+[id="universal_deployer_contract"]
+= Universal Deployer Contract (UDC)
+
+:deploy-syscall: xref:architecture_and_concepts:Smart_Contracts/system-calls-cairo1.adoc#deploy[deploy syscall]
+
+The Universal Deployer Contract (UDC) is a singleton smart contract that wraps the {deploy-syscall} to expose it to any contract that doesn’t implement it, such as account contracts. You can think of it as a standardized generic factory for Starknet contracts.
+
+And since Starknet has no deployment transaction type, it offers a canonical way to deploy smart contracts by following the https://community.starknet.io/t/snip-deployer-contract-interface/2772[standard deployer interface] and emitting a `ContractDeployed` event.
```

Comment:
```suggestion
And since Starknet has no deployment transaction type, it offers a standardized way to deploy smart contracts by following the https://community.starknet.io/t/snip-deployer-contract-interface/2772[standard deployer interface] and emitting a `ContractDeployed` event.
```